lib/browser/api/menu-item-roles.js

```diff
@@ -69,7 +69,7 @@ const roles = {
   },
   redo: {
     label: 'Redo',
-    accelerator: 'Shift+CommandOrControl+Z',
+    accelerator: process.platform === 'win32' ? 'CommandOrControl+Y' : 'Shift+CommandOrControl+Z',
```

Comment:
Purely because I enjoy playing code golf

```js
accelerator: `CommandOrControl+${process.platform === 'win32' ? 'Y' : 'Shift+Z}`
```

**Note:** Don't use this ^^, it isn't very clear what it does 👍  But it is shorter 😆